hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/OzoneClientKeyReadWriteOps.java

```diff
@@ -0,0 +1,258 @@
+    }
+    if (kg == null) {
+      kg = new KeyGeneratorUtil();
+    }
+    runTests(this::readWriteKeys);
+
+    for (int i = 0; i < clientsCount; i++) {
+      if (rpcClients[i] != null) {
+        rpcClients[i].close();
+      }
+    }
+    return null;
+  }
+
+  public void readWriteKeys(long counter) throws Exception {
+    int clientIndex = (int)(counter % clientsCount);
+    OzoneClient client = rpcClients[clientIndex];
+    String operationType = decideReadOrWriteTask();
+    String keyName = getKeyName(operationType, clientIndex);
+    timer.time(() -> {
+      try {
+        switch (operationType) {
+        case readTask:
+          processReadTasks(keyName, client);
+          break;
+        case writeTask:
+          processWriteTasks(keyName, client);
+          break;
+        default:
+          break;
+        }
+      } catch (Exception ex) {
+        LOG.error(ex.getMessage());
+      }
+    });
+  }
+
+  public void processReadTasks(String keyName, OzoneClient client)
+          throws Exception {
+    OzoneBucket ozbk = client.getObjectStore().getVolume(volumeName)
+            .getBucket(bucketName);
+
+    if (readMetadataOnly) {
+      ozbk.getKey(keyName);
+    } else {
+      byte[] data = new byte[writeSizeInBytes];
+      try (OzoneInputStream introStream = ozbk.readKey(keyName)) {
+        introStream.read(data);
+      }
+    }
+  }
+  public void processWriteTasks(String keyName, OzoneClient client)
+          throws Exception {
+    OzoneBucket ozbk = client.getObjectStore().getVolume(volumeName)
+            .getBucket(bucketName);
+
+    try (OzoneOutputStream out = ozbk.createKey(keyName, writeSizeInBytes)) {
+      out.write(keyContent);
+      out.flush();
+    }
+  }
+  public String decideReadOrWriteTask() {
+    if (!isMixWorkload) {
+      if (readRange > 0) {
+        return readTask;
+      } else if (writeRange > 0) {
+        return writeTask;
+      }
+    }
+    //mix workload
+    int tmp = ThreadLocalRandom.current().nextInt(100) + 1; // 1 ~ 100
+    if (tmp < percentageRead) {
+      return readTask;
+    } else {
+      return writeTask;
+    }
+  }
+
+  public String getKeyName(String operationType, int counter) {
+    int startIdx, endIdx;
+    switch (operationType) {
+    case readTask:
+      startIdx = counter * readRange;
+      endIdx = startIdx + readRange;
+      break;
+    case writeTask:
+      startIdx = counter * writeRange;
+      endIdx = startIdx + writeRange;
+      break;
+    default:
+      startIdx = 0;
+      endIdx = 0;
+      break;
+    }
+    StringBuilder keyNameSb = new StringBuilder();
+    int randomIdxWithinRange = ThreadLocalRandom.current().
+            nextInt(endIdx + 1 - startIdx) + startIdx;
+    if (keySorted) {
+      keyNameSb.append(getPrefix()).append(FILE_DIR_SEPARATOR).
+              append(randomIdxWithinRange);
+    } else {
+      keyNameSb.append(getPrefix()).append(FILE_DIR_SEPARATOR).
```

Comment:
Yes! So default behavior is to map the index of key (from 0 to 1M, for example.) to their md5 string, so that the key is not in sorted order in rocksDB. But I left an option here to generate the key name as the index value, so that we could generate sorted key. I'm not sure if we'll test sorted key, I could remove that part if not required!  Let me know!